### PR TITLE
COCICCS-2323 : Enable Certificate Creation option for normal users

### PR DESCRIPTION
**Pull request by NNazeer1**
COCICCS-2323 : Enable Certificate Creation option for normal users

### CODE REVIEW

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java [hunk 1]
```diff
@@ -370,7 +367,17 @@ public CertManagerLogin login(CertManagerLoginRequest certManagerLoginRequest) t
                     CertificateData certDetails = getCertificate(sslCertificateRequest, certManagerLogin);
                     String metadataJson = ControllerUtil.populateSSLCertificateMetadata(sslCertificateRequest,
                             userDetails, token,certDetails);
-                    boolean sslMetaDataCreationStatus = ControllerUtil.createMetadata(metadataJson, token);
+                    
+					boolean sslMetaDataCreationStatus = false;
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java:371]: sslMetaDataCreationStatus  is set to false always?

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/common/SSLCertificateConstants.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
     public static final String VALUE="value";
     public static final String ITEMS="items";
     public static final String SELECTED_ID="selectedId";
-    public static final String SSL_CERT_PATH = "metadata/sslcerts/";
+    public static final String SSL_CERT_PATH = "metadata/sslcerts";
```
Comment (sivanag1974) [inline tvaultapi/src/main/java/com/tmobile/cso/vault/api/common/SSLCertificateConstants.java:26]: Where is "java.util.SplittableRandom;" used which is declared at the top?

Comment (NNazeer1) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/common/SSLCertificateConstants.java:24]: removed the unused import

File: tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java (hunk 1 above)
Comment (NNazeer1) [reply tvaultapi/src/main/java/com/tmobile/cso/vault/api/service/SSLCertificateService.java:371]: here the variable sslMetaDataCreationStatus  is initailizing, below that we are calling the method ControllerUtil.createMetadata(metadataJson, token); to get the proper values for the variable.

**Pull request merged**